modules/InboundEmail/InboundEmail.php

```diff
@@ -3796,7 +3796,7 @@ public function getMessageTextWithUid($uid, $type, $structure, $fullHeader, $cle
         // Bug 50241: can't process <?xml:namespace .../> properly. Strip <?xml ...> tag first.
         $msgPart = preg_replace("/<\?xml[^>]*>/", "", $msgPart);
 
-        return SugarCleaner::cleanHtml($msgPart, false);
+        return SugarCleaner::cleanHtml($msgPart, true);
```

Comment:
same call in function getMessageText() below (in a massive duplicated code part). Shall we change it over there?

Reply:
I have updated the function